ArcGISRuntimeSDKQt_CppSamples/CloudAndPortal/PortalUserInfo/PortalUserInfo.h

```diff
@@ -62,6 +64,7 @@ class PortalUserInfo : public QQuickItem
   void accessChanged();
   void thumbnailUrlChanged();
   void loadErrorMessageChanged();
+  void portalClosed();
```

Comment:
Can you help me understand what this means? There is no concept of "closed" with Portal, so I think it means something else.